magma/wireable.py

```diff
@@ -8,6 +9,9 @@ def wireable(T1, T2):
     Returns true if T1 can be wired to T2
     """
     T1, T2 = magma_type(T1), magma_type(T2)
+    if ((issubclass(T1, UInt) and issubclass(T2, SInt)) or
```

Comment:
what's the difference between `wireable(x, y)` and `x.is_wireable(y)`? It seems there's some fragmentation of this logic. Ok if it's the case, but ideally should document when to use one vs the other (or update names appropriately).

In this case, I tried wiring a uint to an sint and it worked so that seems to be inconsistent with the logic here? Is this only for muxing?

Reply:
They should be consistent, I can change it to call is_wireable on the types (related to Caleb's feedback)